src/workloads/query/multiplanner/BlockingPlanWin.yml

```diff
@@ -0,0 +1,220 @@
+        # It has to be a separate actor because they target different DBs.
+      active: [0]
+      nopInPhasesUpTo: *maxPhase
+      collection: *coll
+
+- Name: CreateDataset
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: InsertTemplate
+    Parameters:
+      database: *db
+      threads: 1
+      active: [1]
+      nopInPhasesUpTo: *maxPhase
+      repeat: 1
+      collection: *coll
+      docCount: *docCount
+      document: {
+        # Genny's random generators appear to be repeatable, so we might as well
+        # make '_id' explicit to make the overall dataset repeatable. Omitting
+        # '_id' would make it auto-generated based on a timestamp.
+        _id: {^Inc: {start: 0}},
+        x1: &distribution {^RandomDouble: {distribution: uniform, min: 0.0, max: 1.0}},
+        x2: *distribution,
+        x3: *distribution,
+        x4: *distribution,
+        x5: *distribution,
+        x6: *distribution,
+        x7: *distribution,
+        x8: *distribution,
+        x9: *distribution,
+        x10: *distribution,
+        x11: *distribution,
+        x12: *distribution,
+        x13: *distribution,
+        x14: *distribution,
+        x15: *distribution,
+        x16: *distribution,
+        x17: *distribution,
+        x18: *distribution,
+        x19: *distribution,
+        x20: *distribution,
+        x21: *distribution,
+        x22: *distribution,
+        x23: *distribution,
+        x24: *distribution,
+        x25: *distribution,
+        x26: *distribution,
+        x27: *distribution,
+        x28: *distribution,
+        x29: *distribution,
+        x30: *distribution,
+        x31: *distribution,
+        x32: *distribution,
+        x33: *distribution,
+        x34: *distribution,
+        x35: *distribution,
+        x36: *distribution,
+        x37: *distribution,
+        x38: *distribution,
+        x39: *distribution,
+        x40: *distribution,
+        x41: *distribution,
+        x42: *distribution,
+        x43: *distribution,
+        x44: *distribution,
+        x45: *distribution,
+        x46: *distribution,
+        x47: *distribution,
+        x48: *distribution,
+        x49: *distribution,
+        x50: *distribution,
+        x51: *distribution,
+        x52: *distribution,
+        x53: *distribution,
+        x54: *distribution,
+        x55: *distribution,
+        x56: *distribution,
+        x57: *distribution,
+        x58: *distribution,
+        x59: *distribution,
+        x60: *distribution,
+        x61: *distribution,
+        x62: *distribution,
+        x63: *distribution
+      }
+      indexes:
+      - keys: {x1: 1}
+        options: {name: index1}
+      - keys: {x2: 1}
+        options: {name: index2}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: SimpleMultiplannerSelectivity0.01
```

Comment:
[nit/optional] Maybe a better name: `MultiplannerPicksBlockingPlanSelectivity0.01` and so on for the rest.